index.js

```diff
@@ -1,3 +1,14 @@
+if(!process.env.now) require('dotenv').config();
+
+const server = require("./server.js");
+
+const port = process.env.now ? 8080 : 4000;
```

Comment:
.now deployment doesn't automatically set its port? it allows you to set the port yourself even when it is building?